src/services/services.ts

```diff
@@ -2912,6 +2919,63 @@ module ts {
 
                 return filteredMembers;
             }
+
+            function isExternalModuleName(token: Node): boolean {
+                if (token && token.parent && token.kind === SyntaxKind.StringLiteral && isInStringOrRegularExpressionOrTemplateLiteral(token)) {

```

Comment:
A few questions:
1. Why check token.parent here, and not in the return expression? You don't seem to use parent here.
2. Why check isInStringOrRegularExpressionOrTemplateLiteral given that you've already determined it's a StringLiteral?